From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Fri, 28 Feb 2025 20:29:23 +0700
Subject: [PATCH 2/2] init.rc: do not change swappiness

2G variants are extremely sensitive to this kind of stuff
just do not interfere with device sided tuning

Signed-off-by: fukiame <fukiame@proton.me>

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 205e83013..abbced100 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1417,7 +1417,6 @@ on property:persist.sys.power_mode_perf=1
     write /proc/sys/kernel/sched_autogroup_enabled 0
     write /proc/sys/vm/dirty_background_ratio 20
     write /proc/sys/vm/dirty_ratio 50
-    write /proc/sys/vm/swappiness 10
     write /sys/block/sda/queue/rotational 0
     write /sys/block/sda/queue/iostats 0
     write /sys/block/sda/queue/add_random 0
@@ -1451,7 +1450,6 @@ on property:persist.sys.power_mode_perf=0
     write /proc/sys/vm/vfs_cache_pressure 200
     write /proc/sys/vm/dirty_background_ratio 3
     write /proc/sys/vm/dirty_ratio 30
-    write /proc/sys/vm/swappiness 60
     write /sys/devices/system/cpu/cpu0/cpufreq/sched_pixel/up_rate_limit_us 2000
     write /sys/devices/system/cpu/cpu0/cpufreq/sched_pixel/down_rate_limit_us 2000
     write /sys/devices/system/cpu/cpu4/cpufreq/sched_pixel/up_rate_limit_us 2000
